codebuild/bin/s2n_codebuild.sh

```diff
@@ -85,8 +85,12 @@ setup_apache_server() {
 run_integration_v2_tests() {
     setup_apache_server
     "$CB_BIN_DIR/install_s2n_head.sh" "$(mktemp -d)"
-    make clean
-    make integrationv2
```

Comment:
How do the binaries get built? Do we assume that the run_unit_tests bit is done first? Or is that handled in the custom target?

Reply:
I believe that's done by install s2n head.